Services/CO.CDP.OrganisationInformation.Persistence/DatabaseOrganisationRepository.cs

```diff
@@ -38,6 +38,13 @@ public async Task<IEnumerable<OrganisationPerson>> FindOrganisationPersons(Guid
         return await context.Set<OrganisationPerson>().FirstOrDefaultAsync(o => o.Organisation.Guid == organisationId && o.Person.Guid == personId);
     }
 
+    public async Task<Person?> FindPersonWithTenant(Guid personId)
+    {
+        return await context.Persons
+            .Include(p => p.Tenants)
+            .FirstOrDefaultAsync(o => o.Guid == personId);
+    }
```

Comment:
Missing test case.

Reply:
Added unit tests to DatabasePersonRepository.cs